mkdocs/__main__.py

```diff
@@ -248,7 +252,9 @@ def cli():
 
 
 @cli.command(name="serve")
-@click.option('-a', '--dev-addr', help=dev_addr_help, metavar='<IP:PORT>')
+@click.option('-a', '--dev-addr', help=dev_addr_help, metavar='IP:PORT')
```

Comment:
So, please don't get me wrong, but IMHO, this is not a very consistent design. You can specify a port via `--dev-addr`, but also via `--port`? That's two ways of achieving the same thing.

Reply:
Either remove `--port` or deprecate and split up `--dev-addrs` into `--host` and `--port`. I'm not sure what you're aiming here for however, because `--dev-addr` is working perfectly fine.

Reply:
I would find a `-p` option really convenient. I literally *never* change the default host, and if I had to, it would probably be in a container setup where I would set it to 0.0.0.0 and never change it again. Whereas in (rare) cases where I serve multiple sites, I need to change the port, and having to re-specify the default host 127.0.0.1 is a bit annoying.

So I wouldn't mind if `--dev-addr` was deprecated in favor of two `--host` and `--port` options. However I'm not the only MkDocs user and I fear it would be quite disruptive to deprecate `--dev-addr` for, again, rarely happening situations. Maybe they could both (dev-attr and host+port) live together, forever, as mutually exclusive options?